docs/docs/ecosystem/rfp/intention-engine.md

```diff
@@ -0,0 +1,10 @@
+# Request for Proposal: Intention Engine
+
+## Introduction:
+
+Composable Finance is seeking proposals from qualified blockchain developers 
+to design and integrate a **Proof-of-Concept** for a custom intention engine for multi-chain usage 
+using [Cross-Chain Virtual Machine(XCVM)]. 
+The goal of this project is to allow projects to create and maintain request, propose solution and pick solution to solve cross chain execution in most efficient manner.
+
+[Cross-Chain Virtual Machine(XCVM)]: https://docs.composable.finance/products/xcvm
```

Comment:
```suggestion
using [Cross-Chain Virtual Machine (XCVM)]. 
The goal of this project is to allow projects to create and maintain request, propose solution and pick solution to solve cross chain execution in most efficient manner.

[Cross-Chain Virtual Machine (XCVM)]: https://docs.composable.finance/products/xcvm
```

Reply:
yeah, tbh i think we should think as pm or final power user and be clear mechanics/requirements. 

like cow protocol.

Reply:
and than only go into details of json/rust/protobuf/vm/contract. all these are low level agains protocol